hbase-server/src/main/java/org/apache/hadoop/hbase/executor/ExecutorService.java

```diff
@@ -127,25 +127,14 @@ public ThreadPoolExecutor getExecutorThreadPool(final ExecutorType type) {
     return getExecutor(type).getThreadPoolExecutor();
   }
 
-  public void startExecutorService(final ExecutorType type, final ExecutorConfig config) {
-    String name = type.getExecutorName(this.servername);
-    if (isExecutorServiceRunning(name)) {
-      LOG.debug("Executor service {} already running on {}", this,
-          this.servername);
-      return;
-    }
-    startExecutorService(config.setName(name));
-  }
-
   /**
    * Initialize the executor lazily, Note if an executor need to be initialized lazily, then all
```

Comment:
I'm not sure I understood this comment. In what sense this is lazier compared to startExecutorService()? Wanted to get rid of this but checking here first.

Reply:
I didn't get this either, method name also has lazy word.